Resources/Prototypes/_Goobstation/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -0,0 +1,42 @@
+- type: entity
+  parent: [ ClothingOuterHardsuitBase, BaseSyndicateContraband ]
+  id: ClothingOuterHardsuitCybersunStealth
+  suffix: stealth
```

Saying the suit with "stealth" in its name is a stealth suit isn't really what suffixes are for. That's also player/admin facing so comment it out please.
```suggestion
  # suffix: stealth
```